packages/manager/modules/iam/src/translations/Messages_fr_FR.json

```diff
@@ -10,5 +10,88 @@
   "iam_guide_description_users": "Découvrez comment ajouter des utilisateurs depuis votre compte OVHcloud.",
   "iam_guide_iam": "Créer et gérer des politiques",
   "iam_guide_samlsso": "Activer la fédération SAML",
-  "iam_guide_users": "Créer et gérer des utilisateurs"
+  "iam_guide_users": "Créer et gérer des utilisateurs",
+  "iam_resource_type_account": "Mon compte",
+  "iam_resource_type_alldom": "Domain packs",
+  "iam_resource_type_analytics": "Analytics",
+  "iam_resource_type_anthos": "Hosted Private Cloud powered by Anthos",
+  "iam_resource_type_cdn": "CDN",
+  "iam_resource_type_cloudDB": "Enterprise Cloud Database",
+  "iam_resource_type_cspReseller": "Revendeur CSP",
+  "iam_resource_type_custom": "Actions ajoutées manuellement",
+  "iam_resource_type_dedicatedCeph": "Cloud Disk Array",
+  "iam_resource_type_dedicatedNVMeoF": "NVMeoF dédié",
+  "iam_resource_type_dedicatedServer": "Serveur Dédié (Baremetal)",
+  "iam_resource_type_dnsZone": "Zone DNS",
+  "iam_resource_type_dockerRegistry": "Registre Docker",
+  "iam_resource_type_dockerStack": "Decker Stack",
+  "iam_resource_type_domain": "Domain name",
+  "iam_resource_type_emailDomain": "Emails Domain",
+  "iam_resource_type_emailMxplan": "MX Plan Email",
+  "iam_resource_type_emailPro": "Email Pro",
+  "iam_resource_type_freefax": "Fax",
+  "iam_resource_type_hadoopClusterAccount": "Analytics Data Platform",
+  "iam_resource_type_horizonView": "Horizon",
+  "iam_resource_type_hostingOption": "Options Hébergement",
+  "iam_resource_type_housing": "Housing",
```

Comment:
Maybe we would say `Hosting` ?

Reply:
Actually the type is `housing`, hence the "translation"